autoload/health/nvim_lsp.vim

```diff
@@ -0,0 +1,8 @@
+let s:FILE_DIR = fnamemodify(expand("<sfile>"), ':h') . '/'
+let s:FILE_DIR = substitute(s:FILE_DIR, '\\', '/', 'g')
+
+function! health#nvim_lsp#check()
+	call health#report_start('Checking language server protocol configuration')
+	execute 'luafile '.s:FILE_DIR.'check.lua'
```

Comment:
if `autoload/health/check.lua` were moved to `lua/nvim_lsp/health.lua`, then this could be:
```suggestion
	require('nvim_lsp.health').nvim_lsp_check_health()
```

And then the workaround in `.luacheckrc` would not be needed.

Is there a reason for the current approach?

Reply:
you are right, thats what I've done afterwards but I haven't had the opportunity to test it yet. my system is rebuilding.